src/output-json.h

```diff
@@ -41,8 +41,8 @@ int OutputJSONMemBufferCallback(const char *str, size_t size, void *data);
 
 void CreateJSONFlowId(json_t *js, const Flow *f);
 void JsonTcpFlags(uint8_t flags, json_t *js);
-json_t *CreateJSONHeader(const Packet *p, int direction_sensative, const char *event_type);
-json_t *CreateJSONHeaderWithTxId(const Packet *p, int direction_sensitive, const char *event_type, uint64_t tx_id);
+json_t *CreateJSONHeader(const Packet *p, int direction_sensative, const char *event_type, LogFileCtx *file_ctx);
```

Comment:
Something like that:
https://github.com/regit/suricata/commit/d31938bd4294d13f26e5bff88627586b10faa2f4#diff-e2a291d82d2cff36c51007fafaa79b86R434